aldryn_newsblog/models.py

```diff
@@ -251,7 +251,7 @@ def save(self, *args, **kwargs):
             all_slugs = Article.objects.language(lang).exclude(
                 pk=self.pk).values_list('translations__slug', flat=True)
             for slug in all_slugs:
-                if slug and slug.startswith(self.title) or slug.startswith(self.slug):
+                if slug and slug.startswith((self.title, self.slug)):
```

Comment:
I'm not sure if `self.title` is valid here. Shouldn't `self.slugify(self.title)` be used?